### PR TITLE
Read on create only returns null instead of hiding field.

### PR DESCRIPTION
**Pull request by hibooboo2**

### CODE REVIEW

**Comment by ibuildthecloud** [comment]
@hibooboo2 tests are failing

**Comment by ibuildthecloud** [comment]
@hibooboo2 I don't quiet understand.  It seems like the old behavior is correct.  Why do we want to return null as opposed to not sending the field?  Is this causing a downstream issue?  @vincent99 Did you complain about this one?

**Comment by hibooboo2** [comment]
It is causing isssues yes.

File: code/framework/java-server/src/main/java/io/github/ibuildthecloud/gdapi/model/impl/WrappedResource.java [hunk 1]
```diff
@@ -68,13 +68,16 @@ protected void init() {
...
             Field field = entry.getValue();
-            if (!field.isIncludeInList() || (!StringUtils.equalsIgnoreCase(method, "post") && field.isReadOnCreateOnly())) {
+            if (!field.isIncludeInList()) {
                 continue;
             }
             Object value = additionalFields.remove(name);
             if (value == null) {
                 value = field.getValue(obj);
             }
+            if (!StringUtils.equalsIgnoreCase(method, "post") && field.isReadOnCreateOnly()){
```
Comment (ibuildthecloud) [inline code/framework/java-server/src/main/java/io/github/ibuildthecloud/gdapi/model/impl/WrappedResource.java]: There is a constant or method somewhere for "post"

**Comment by ibuildthecloud** [comment]
sounds like there was discussion amongst @cjellick and @vincent99 regarding this.  I'm okay with the change then.

**Pull request merged**